pennylane/ftqc/parametric_midmeasure.py

```diff
@@ -389,10 +391,20 @@ def _flatten(self):
     @property
     def hash(self):
         """int: Returns an integer hash uniquely representing the measurement process"""
+        if is_abstract(self.angle):  # pragma: no cover
+            # no unique value from tracer to values, hash based on object string
+            param_hash = hashlib.sha256(str(self).encode()).digest()
```

Comment:
Just want to confirm if we want to convert the `self` object or `self.angle` to a `str` object?

Reply:
There's no better solution here when dealing with a jax tracer, as there's no numeric value. Using the string offers us the unique memory address as part of the string, which is treated correctly with the buffer protocol of the hasher.